src/gemini/index.ts

```diff
@@ -0,0 +1,80 @@
+import { FirebaseError } from "../error";
+import { logger } from "../logger";
+import * as fs from "fs";
+import * as path from "path";
+import { promptOnce } from "../prompt";
+import { execSync } from "child_process";
+import * as clc from "colorette";
+
+const { GoogleGenerativeAI } = require("@google/generative-ai");
+
+// TODO(christhompson): Do we have an endpoint that's open and doesn't require a project w/ billing?
```

Comment:
Pretty sure everything is going to require an API Key. You can get one from AIStudio tho pretty easily.